sunfish.go

```diff
@@ -15,10 +17,29 @@ type Sunfish struct {
 
 	Router *mux.Router
 	Routes []Route
+
+	logger  *log.Logger
+	logFile *os.File
+}
+
+func NewLogger(sf *Sunfish, logDir string) {
```

Comment:
fine for now, want to merge this b/c it's stale.

But the typical way to do this would be 

`func (sf *Sunfish) initLogger() {`, and then logDir would be a variable in the sunfish object.